editions/2023/fa/0x10-api-security-risks.md

```diff
@@ -0,0 +1,37 @@
+
+
+
+# RISK ریسک های امنیتی API
+========
```

Comment:
Not required: this is an [alternate syntax for `#`](https://www.markdownguide.org/basic-syntax/#alternate-syntax).

Please review the translation source for issues likes this one.